Dockerfile

```diff
@@ -3,8 +3,7 @@ FROM python:3.7
 WORKDIR /app
 
 # copy repo contents
-COPY setup.py README.md ./
-COPY optimade ./optimade
+ADD ./  /app/
```

Comment:
I would not do this, but instead keep the COPY statements to only have the absolute minimum/essential within the docker container.

Reply:
Another one of those setuptools_scm hacks. I so badly wanted that to work out. :(